### ISSUE TITLE
aquamacs doesn't respect `aquamacs-scratch-file`

### ISSUE THREAD

**Issue by nega0** [aquamacs-emacs/aquamacs-emacs#158]
Aquamacs will query to save the `*scratch*` buffer when `aquamacs-scratch-file` is set to `nil`. This occurs with normal invocation, and with `-q`.

**Comment by itollefsen** [comment]
I can't remember it ever asking me to save the scratch buffer - I thought it just did it automatically. Anyway, I'm running 3.5 on Mojave and have something like this in my .emacs file:
```
(unless window-system
  (custom-set-variables '(aquamacs-scratch-file nil)))
```
The reason for that is that I noticed it had trouble reading the scratch file when invoked with `-nw` from a terminal. But I'm not interested in a scratch file if I do that.

Setting `aquamacs-scratch-file` to `nil` works for me in the sense that it doesn't even load it (so it certainly won't ask about saving it).

**Comment by treese** [comment]
Fixed in 3962e5843ae9131e753a86209ff9871caa011997, I think. The variable `buffer-offer-save` was set in `*scratch*` by `aquamacs-turn-on-buffer-offer-save-in-scratch` even when `aquamacs-scratch-file` was set to nil.
